src/helper.js

```diff
@@ -520,6 +526,21 @@ function createProperty(property = {}) {
   return undefined;
 }
 
+function getIpAddress(request) {
+  const IP_ADDRESS = /\b(?:\d{1,3}\.){3}\d{1,3}\b/g;
+
+  if (
```

Comment:
Let's add destructures for `context` and `geo` and replace this `if` with a return like this:
`return typeof geo.ipAddress === "string" && IP_ADDRESS.test(geo.ipAddress) ? geo.ipAddress : undefined`